extras/indexing/src/main/java/mvm/rya/indexing/accumulo/ConfigUtils.java

```diff
@@ -67,6 +67,10 @@
 
 /**
  * A set of configuration utils to read a Hadoop {@link Configuration} object and create Cloudbase/Accumulo objects.
+ * Soon will deprecated this class.  Use installer for the set methods, use RyaDetails for the get methods. 
```

Comment:
"This class will soon be deprecated, replaced with {@link installer} and {@link RyaDetails}....."